src/Microsoft.DotNet.Darc/src/DarcLib/VirtualMonoRepo/VmrInitializer.cs

```diff
@@ -59,11 +59,13 @@ public async Task InitializeVmr(SourceMapping mapping, string? targetRevision, C
         await ApplyPatch(mapping, patchPath, cancellationToken);
         cancellationToken.ThrowIfCancellationRequested();
         await TagRepo(mapping, commit.Id.Sha);
+        cancellationToken.ThrowIfCancellationRequested();
+        await ApplyVmrPatches(mapping, cancellationToken);
 
-        var description = PrepareCommitMessage(InitializationCommitMessage, mapping, null, commit.Id.Sha, null);
+        var message = PrepareCommitMessage(InitializationCommitMessage, mapping, oldSha: null, commit.Id.Sha, null);
```

Comment:
Maybe the last param could also have its name specified as `oldSha` now has?

Reply:
I've made the nullable parameters optional to make this better